### DISCUSSION TITLE
Open Source Contribution Guides

### DISCUSSION THREAD

**Discussion by Valcoholics** [public-assembly/.github#3] [Ideas]
## 👋 Want to contribute to Public Assembly?

If you're passionate about contributing to Public Assembly, [Open Source Guide](https://opensource.guide/how-to-contribute/) can provide you with a great opportunity to get involved. Open source projects are a collaborative effort where individuals can contribute their skills and knowledge towards a common goal, which can ultimately benefit the public.

The guides were created and are curated by GitHub, along with input from outside community reviewers, but they are not exclusive to GitHub products. 

### Find support for your project ideas: 
[Submitting a proposal](https://nouns.build/dao/0xd2e7684cf3e2511cc3b4538bb2885dc206583076/78?tab=activity) is a crucial step in ensuring that your contributions to an open source project are properly recognized and supported. When submitting a proposal, be sure to include a detailed description of the work you plan to do, along with a timeline and estimated budget. 

You can find more help on submissions in the [Public Assembly Forum.](https://forum.public---assembly.com/) Make sure to introduce yourself and outline any relevant experience you have and highlight the impact your work will have on projects.

It's important to note project proposal approvals are governed by Public Assembly token holders and sponsorship may not always be available, but don't let that discourage you from contributing to the project. There are still many ways to contribute, such as by reporting bugs, creating documentation, or helping with community management.

**Comment by projectmehari** [comment]
Thanks for setting this up Val! 

One of the easiest ways to contribute is by creating a Wiki for any ongoing Public Assembly project. I have already started this process with [[Assembly Press](https://github.com/public-assembly/AssemblyPress/wiki/Glossary-and-key-concepts)](https://github.com/public-assembly/AssemblyPress/wiki/Glossary-and-key-concepts) by adding a glossary of key concepts and terms.

I encourage anyone who wants to become more involved to give it a try as well.
